modelforge/train/training.py

```diff
@@ -882,11 +880,11 @@ def read_config_and_train(
         number_of_nodes,
     )
 
-    perform_training(
-        potential_parameters=potential_parameters,
-        training_parameters=training_parameters,
-        dataset_parameters=dataset_parameters,
-        runtime_parameters=runtime_parameters,
+    return perform_training(
+        potential_config=potential_config,
+        training_config=training_config,
+        dataset_config=dataset_config,
+        runtime_config=runtime_config,
     )
```

Comment:
I also fixed this in the PR I'm working on.  I changed names for some reason when I decided to split this into a separate function (we have no testing of the entire function call which might be problematic).

Reply:
I added a test in this PR for this particular function!